lib/isodoc/function/to_word_html.rb

```diff
@@ -33,7 +35,7 @@ def tmpimagedir
     end
 
     def rel_tmpimagedir
-      @filename.sub(%r{^.*/([^/]+)$}, "\\1") + tmpimagedir_suffix
+      Pathname.new(@filename).basename.to_s + tmpimagedir_suffix
```

Comment:
Sounds like `@filename` should be a `Pathname`?

Reply:
Yep ideally, yes, but this will lead to refactoring

@ronaldtse should I do it in the scope of this PR?

Reply:
New PR then.